### PR TITLE
Updated composer depedencies to work with Codeception 5

### PR DESCRIPTION
**Pull request by developedsoftware**
| Q             | A
| ------------- | ---
| Is bugfix?    | o
| New feature?  | no
| Breaks BC?    | no
| Tests pass?   | unknown

### CODE REVIEW

**Comment by developedsoftware** [comment]
Not sure if this can even be done yet - as codception 5 requires PHP 8. 

Yii2 I assume will support PHP 7 for quite some time...should I just close this?

**Comment by samdark** [comment]
@developedsoftware I think it's possible to allow both. Is there a stable release already?

**Comment by developedsoftware** [comment]
Yes I believe so. I will look to do that ! Thanks 

I always think composer is more linear than it is !

**Comment by developedsoftware** [comment]
OK this should be good to go now!

**Comment by developedsoftware** [comment]
Requesting approval if possible.

**Comment by bizley** [comment]
I'm worried about all the dev dependencies here...

File: codeception.yml [hunk 1]
```diff
@@ -4,6 +4,6 @@ include:
     - frontend
     - backend
 paths:
-    log: console/runtime/logs
+    output: console/runtime/output
```
Comment (My6UoT9) [inline codeception.yml:7]: is this needed, log seems just fine - and could potentially break some existing stuff?

Comment (developedsoftware) [reply codeception.yml:7]: log is dropped in favour of output in codeception 2.3 so there should not be any breaking changes (all my tests passed locally) 

See here https://github.com/Codeception/Codeception/issues/4688#issuecomment-354017094

My understanding was that log was transposed to output in the background for some time (for backward compatibilty before being dropped altogether which causes it to now error)

Original issue -> https://github.com/Codeception/Codeception/issues/4688

**Comment by bizley** [comment]
I mean `^5.0.0-alpha2` or `9.5.x-dev` or `dev-codecept5`

**Comment by developedsoftware** [comment]
> I mean `^5.0.0-alpha2` or `9.5.x-dev` or `dev-codecept5`

I dont follow?

`9.5.x-dev` is for phpunit/phpunit? 

`dev-codecept5` is for [codeception/module-yii2](https://github.com/Codeception/module-yii2/tree/codecept5) which requires `"codeception/codeception": "^5.0.0-alpha2"`

which is also the requirement for `codeception/codeception` in this module `^5.0.0-alpha2`

So to me that all checks out OK? Can you clarify?

**Comment by developedsoftware** [comment]
> I'm worried about all the dev dependencies here...

I know what you mean when you said you worry about them dev dependencies. Yes I agree it would be nice to have a stable releases but there arent any so I am forced to use the dev branches.

**Review by samdark — APPROVED** [review]

**Comment by samdark** [comment]
Thanks.

**Pull request merged**